packages/mcp/src/index.ts

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env node
+import { StdioServerTransport } from "@modelcontextprotocol/sdk/server/stdio.js";
+import chalk from "chalk";
+
+import { RecallAgentToolkit } from "@recallnet/agent-toolkit/mcp";
+import {
+  Configuration,
+  Permission,
+  Resource,
+} from "@recallnet/agent-toolkit/shared";
+
+/**
+ * Options for the Recall MCP server
+ * @param tools - The tools to enable
+ * @param privateKey - The private key to use
+ * @param network - The Recall network to use: `testnet` or `localnet` (defaults to `testnet`)
+ */
+type Options = {
+  tools?: string[];
+  privateKey?: string;
+  network?: string;
+};
+
+// The server will accept flags for a `--private-key` (hex string), `--tools` (comma separated),
+// and `--network` (e.g., `testnet` or `localnet`)
+const ACCEPTED_ARGS = ["private-key", "tools", "network"];
+
+// Note: since we require a private key, we only expose both read and write tools
+const ACCEPTED_TOOLS = [
+  "account.read",
+  "account.write",
+  "bucket.read",
+  "bucket.write",
+];
+
+// eslint-disable-next-line turbo/no-undeclared-env-vars
```

Comment:
You _could_ extend process.env so that you get type checking here if you wanted. No problem if not though.

Reply:
kk i'll try that. the other option is to just add them to the [turbo config](https://github.com/recallnet/js-recall/blob/ee0fa0f26060a522a09c2df8dff7525c556cc3ed/turbo.json#L4), but those seem to be application specific (e.g., Portal, Faucet) and not meant for what we're doing there.